website/src/index.css

```diff
@@ -151,8 +151,8 @@ body {
     hsla(203, 11%, 15%, 0.4);
   color: #aaa !important;
 }
-.bg-clip-text {
+ .heading {
   background: -webkit-linear-gradient(#ff8533, #ff4da6);
-  background-clip: border-box;
+   -webkit-background-clip: text;
```

Comment:
Remove this line

Reply:
I can't  remove  the webkit-background-clip : text ; 
Line the gradient colour  is in the background if I remove this the text  wouldn't be visible ![20210920_232743.jpg](https://user-images.githubusercontent.com/75156752/134050956-22b6fd01-d290-4fab-a27e-dfdef88f38e1.jpg)

Reply:
I've tested it, and using the following:

```css
background: -webkit-linear-gradient(#ff8533, #ff4da6);
-webkit-text-fill-color: transparent;
```

with the class `.bg-clip-text` should work successfully.

You now have the class `.bg-clip-text`. Please update the CSS to the following:

```css
.heading {
  background: -webkit-linear-gradient(#ff8533, #ff4da6);
  -webkit-text-fill-color: transparent;
}
```